CRM/Membershipextrasdata/Utils.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class CRM_Membershipextrasdata_Utils {
+
```

Comment:
Perhaps it is good to add the comment to explain what this function does?

Reply:
Added PHPDoc comment blocks whenever possible.